vm/class.go

```diff
@@ -287,6 +305,28 @@ func createBaseClass(className string) *BaseClass {
 // builtinCommonInstanceMethods is a collection of common instance methods used by Class
 var builtinCommonInstanceMethods = []*BuiltInMethodObject{
 	{
+		// Loads the given Goby library name without extension (mainly for modules), returning `true`
+		// if successful and `false` if the feature is already loaded.
+		//
+		// Currently, only the following embedded Goby libraries are targeted:
+		//
+		// - "file"
+		//
+		// - "net/http"
+		//
+		// - "net/simple_server"
+		//
+		// - "uri"
+		//
```

Comment:
The blank line is needed for generating HTML: they would be like `-point -point -point -
 point` if  no blank.